Automattic/jetpack
# Move enhanced open graph from modules/publicize to modules/

#### Changes proposed in this Pull Request:
Remove the enhanced-open-graph.php file out of the Publicize module and move it into the modules directory of the Jetpack plugin. 

This PR came [out as a result of another PR ](https://github.com/Automattic/jetpack/pull/23786#discussion_r842889187)I am working on where we make Jetpack use the Publicize package, from the packages folder. It was decided that the enhanced-open-graph.php file shouldn't live in the Publicize module and be a file of its own. 

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
 
 - Create a post and add a gallery of photos to it using the /gallery block. 
 - Publish the post, go the post detail view and view the source code of the page. You will see the open graph tags added like this. 

  
<img width="1372" alt="Screenshot 2022-04-12 at 3 45 57 PM" src="https://user-images.githubusercontent.com/6594561/162939003-5201ec3b-798c-4ceb-9ef8-b9fa08608b1d.png">

    
 - Deactivate the 'publicize' and 'sharing' modules, from wp-admin. You can click on the modules at the bottom of the screen to manage your modules. 
 
 
<img width="856" alt="Screenshot 2022-04-12 at 3 47 07 PM" src="https://user-images.githubusercontent.com/6594561/162939200-be189ba2-8ba9-46f9-be6c-2ba90bd21d82.png">

 
- Go back to your post detail view, and view the page source again. The open graph tags won't be added.
